lambdas/indexer/app.py

```diff
@@ -77,6 +78,7 @@ def post_notification():
     """
     Receive a notification event and either queue it for asynchronous indexing or process it synchronously.
     """
+    validate(current_request=app.current_request)
```

Comment:
Change the import such that this reads

```suggestion
    hmac.validate(current_request=app.current_request)
```